logdna/logdna.py

```diff
@@ -50,32 +54,33 @@ def __init__(self, key, options={}):
         self.setLevel(logging.DEBUG)
         self.lock = threading.RLock()
 
-        self.session = requests.Session()
-        retry = Retry(connect=self.connection_retries, backoff_factor=self.retry_backoff_factor)
-        adapter = HTTPAdapter(max_retries=retry)
-        self.session.mount('https://', adapter)
-
     def buffer_log(self, message):
         if message and message['line']:
             if len(message['line']) > self.max_length:
                 message['line'] = message['line'][:self.max_length] + ' (cut off, too long...)'
                 if self.verbose in ['true', 'debug', 'd']:
-                    logger.debug('Line was longer than %s chars and was truncated.', str(self.max_length))
+                    print(f'Line was longer than {self.max_length} chars and was truncated.')
```

Comment:
I think we should go back to logger, but will leave up to you. Note though that this makes this library specifically Python 3 only and that we'll have to communicate that to all customers who use it.

Reply:
I prefer to print out to the terminal because logger does not really work well with saving everything in file. I don't understand why it will work with python3 only?

Reply:
> I don't understand why it will work with python3 only?

Python 2 and Python 3 have different `print` formats and aren't compatible. Your `print` function call won't work on Python 2.